src/app/components/tag/TagInput.js

```diff
@@ -106,8 +109,8 @@ class TagInput extends React.Component {
             onKeyPress={this.handleKeyPress.bind(this)}
             placeholder={this.props.intl.formatMessage(messages.search)}
           />
-          <Button
-            style={{ marginLeft: 'auto' }}
+          <Box ml="auto">
+          <Button 
```

Comment:
Please remove trailing space in this line.
Also please indent the `<Button>` component correctly now that it is a child of `<Box>`.